extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -409,11 +409,11 @@ export class AmpA4A extends AMP.BaseElement {
                         if (isValid) {
                           return creativeParts.creative;
                         }
-                        return Promise.reject(
-                            'Key failed to validate creative\'s signature.');
+                        throw new Error("Key failed to validate creative's " +
```

Comment:
+1  We know that we need to clean up a lot of the logic in here and clarify what's a real error, what just needs a console message, etc.  But that's a bigger and more unified effort than this change.